insights/specs/default.py

```diff
@@ -432,6 +432,7 @@ def httpd_cmd(broker):
     modinfo_veth = simple_command("/sbin/modinfo veth")
     modinfo_vmxnet3 = simple_command("/sbin/modinfo vmxnet3")
     modprobe = glob_file(["/etc/modprobe.conf", "/etc/modprobe.d/*.conf"])
+    mokutil_sbstate = simple_command("mokutil --sb-state")
```

Comment:
@jobselko would you please add the path for this command, for example `/usr/bin/mokutil --sb-state` or `/bin/modutil --sb-state`?  It need to match the JSON spec.